docs/custom-flows/error-handling.mdx

```diff
@@ -13,31 +13,67 @@ A user is attempting to sign up with your application, but they are attempting t
 
 <CodeBlockTabs type="framework" options={["React", "JavaScript"]}>
 ```jsx
-import { useState } from "react";
+import { useState, SyntheticEvent } from "react";
 import { useSignUp } from "@clerk/clerk-react";
 
 function Example() {
-  const [error, setError] = useState(null);
-
-  const { signUp } = useSignUp();
-
-  const signUpUser = async () => {
-    const response = await signUp
-      ?.create({
-        identifier: "example@email.com",
-        password: "Tester01!",
-      })
-      .then((res) => console.log("response", res))
-      .catch((err) => setError(err.errors[0].message));
+  const [emailAddress, setEmailAddress] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState<any>();
+
+  const { isLoaded, signUp, setActive } = useSignUp();
+
+  const attemptSignUp = async (e: SyntheticEvent) => {
+    e.preventDefault();
+
+    // clear error on form submit
+    setError(undefined);
```

Comment:
Can you elaborate on what this is doing. We just need to be clear for documentation.
Likely, clearing a previously set error from a form submission attempt?